### PR TITLE
FIX: ModalFooter box-shadow, Close position

### PR DESCRIPTION
**Pull request by wEprous**
Fixed box-shadow in ModalFooter when the Modal is fully scrolled.
Fixed position of CloseContainer during initial animation on Mobile.<br/><br/><br/><url>LiveURL: https://orbit-components-fix-modal-footer-boxshadow.surge.sh</url>

### CODE REVIEW

**Comment by tomashapl** [comment]
@wEprous Can you please rebase and resolve conflicts before CR ? 🙏

**Comment by wEprous** [comment]
Done! 🙂

File: src/Modal/index.js [hunk 1]
```diff
@@ -188,9 +194,9 @@ ModalWrapperContent.defaultProps = {
   theme: defaultTokens,
 };
 
-const CloseElement = ({ onClose }: CloseElementType) => (
+const CloseElement = ({ onClose, closable }: CloseElementType) => (
   <CloseContainer>
-    <ButtonLink onClick={onClose} size="normal" icon={<Close />} transparent />
+    {closable && <ButtonLink onClick={onClose} size="normal" icon={<Close />} transparent />}
```
Comment (tomashapl) [inline src/Modal/index.js]: please this condition to `319` line before render the whole `CloseElement`

File: src/Modal/index.js [hunk 2]
```diff
@@ -241,6 +249,10 @@ class Modal extends React.PureComponent<Props, State> {
     if (ev.target instanceof HTMLDivElement) {
       this.setState({
         scrolled: ev.target.scrollTop >= this.offset,
+        fixedClose: ev.target.scrollTop >= 1,
+        fullyScrolled:
+          this.props.fixedFooter &&
+          ev.target.scrollTop >= ev.target.scrollHeight - ev.target.clientHeight - 10,
```
Comment (tomashapl) [inline src/Modal/index.js]: what is `-10` ?

File: src/Modal/index.js (hunk 1 above)
Comment (wEprous) [reply src/Modal/index.js]: You can't the `CloseContainer` is used as background for the mobile header (when scrolled).

Comment (tomashapl) [reply src/Modal/index.js]: OK, but it's little bit confusing.

File: src/Modal/index.js (hunk 2 above)
Comment (tomashapl) [reply src/Modal/index.js]: Ok, so please can you add some comments to code ?

File: src/Modal/index.js (hunk 1 above)
Comment (wEprous) [reply src/Modal/index.js]: We will see how it will looks like after another update :)

File: src/Modal/index.js (hunk 2 above)
Comment (wEprous) [reply src/Modal/index.js]: OK 👌

**Pull request merged**